src/app/lib/utilities/preprocessor/rules/filterStraplines.test.js

```diff
@@ -0,0 +1,194 @@
+import filterStraplines from './filterStraplines';
+
+describe('strapline rules', () => {
+  it('should keep groups with a strapline', () => {
+    const data = {
+      content: {
+        groups: [
+          {
+            items: [
+              {
+                headlines: {
+                  headline: 'This is a headline',
+                },
+              },
+              {
+                headlines: {
+                  headline: 'This is another headline',
+                },
+              },
+            ],
+            strapline: {
+              name: 'Strapline name',
+            },
+          },
+        ],
+      },
+    };
+
+    const expected = {
+      content: {
+        groups: [
+          {
+            items: [
+              {
+                headlines: {
+                  headline: 'This is a headline',
+                },
+              },
+              {
+                headlines: {
+                  headline: 'This is another headline',
+                },
+              },
+            ],
+            strapline: {
+              name: 'Strapline name',
+            },
+          },
+        ],
+      },
+    };
+
+    expect(filterStraplines(data)).toEqual(expected);
```

Comment:
Since `data` and `expected` are the same in this case - maybe this line can just be ```expect(filterStraplines(data)).toEqual(data);``` and remove `expected`

Reply:
Thanks for the review, fixed it:)